olympics.md

```diff
@@ -0,0 +1,25 @@
+--
```

Comment:
I believe you need 3 hyphens like here https://jekyllrb.com/docs/front-matter/ or see example page here https://raw.githubusercontent.com/datasets/awesome-data/refs/heads/fix/outdated-links/air-pollution.md